programs/mango-v4/src/accounts_ix/openbook_v2_cancel_all_orders.rs

```diff
@@ -0,0 +1,47 @@
+use anchor_lang::prelude::*;
+
+use crate::error::*;
+use crate::state::*;
+use openbook_v2::{
+    program::OpenbookV2,
+    state::{BookSide, Market, OpenOrdersAccount},
+};
+
+#[derive(Accounts)]
+pub struct OpenbookV2CancelAllOrders<'info> {
+    #[account(
+        constraint = group.load()?.is_ix_enabled(IxGate::OpenbookV2CancelAllOrders) @ MangoError::IxIsDisabled,
+    )]
+    pub group: AccountLoader<'info, Group>,
+
+    #[account(
+        has_one = group,
+        constraint = account.load()?.is_operational() @ MangoError::AccountIsFrozen
+        // owner is checked at #1
+    )]
+    pub account: AccountLoader<'info, MangoAccountFixed>,
+    pub owner: Signer<'info>,
+
+    #[account(mut)]
+    pub open_orders: AccountLoader<'info, OpenOrdersAccount>,
```

Comment:
document how we know that this open_orders is the right one

Reply:
Will add a check on market.